.github/workflows/dist.yml

```diff
@@ -39,6 +39,8 @@ jobs:
       with:
         python-version: ${{ matrix.python_version }}
         architecture: ${{ matrix.architecture }}
+    - uses: robotpy/build-actions/setup-cxx@gcc-8
+      if: ${{ matrix.os == 'ubuntu-18.04' }}
```

Comment:
Hm. I wonder if we should just use a container instead?

Reply:
Probably a good idea, but I'm not sure whether we can fit containers into the workflow without duplicating the entire build job.